package.json

```diff
@@ -12,10 +12,13 @@
     "postinstall": "cd packages/react-instantsearch && npm install",
     "doctoc": "doctoc --maxlevel 1 README.md CONTRIBUTING.md",
     "test": "(cd packages/react-instantsearch && npm test) && npm run lint && NODE_ENV=production DOCS_MOUNT_POINT=/instantsearch.js/react/ DOCS_DIST=docs/react/ npm run docs:build && NODE_ENV=production npm run build",
-    "dev": "cd packages/react-instantsearch && npm run dev"
+    "dev": "cd packages/react-instantsearch && npm run dev",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Well we can have a try, but I don’t want to add something because it fits my workflow but it somehow duplicates stuff :slightly_smiling_face: Maybe I can fill it while working for the default theme stories, and you guys tell me what you think of it.